h/views/groups.py

```diff
@@ -114,14 +114,15 @@ def _update_group(self, appstruct):
              request_method='GET',
              renderer='h:templates/groups/share.html.jinja2',
              effective_principals=security.Authenticated,
+             has_feature_flag=not_('search_page'),
              has_permission='read')
 def read(group, request):
     """Group view for logged-in users."""
-    _check_slug(group, request)
+    check_slug(group, request)
```

Comment:
I'm not super happy about changing an `h.views.groups` private helper function into a public one and having `h.views.activity` import it but it's a quick way to get the job done and I want to refactor these views more later so that the group_read views aren't spread across two separate files.